application/views/main/dashboard/home.php

```diff
@@ -31,7 +31,7 @@
 			<tr valign="top">
 				<td><?php echo tr('Gammu version');?></td>
 				<td>:</td>
-				<td><?php echo  filter_data(htmlentities($this->Kalkun_model->get_gammu_info('gammu_version')->row('Client')), ENT_QUOTES); ?></td>
+				<td><?php echo  filter_data(htmlentities($this->Kalkun_model->get_gammu_info('gammu_version')->row('Client') ?? ''), ENT_QUOTES); ?></td>
```

Comment:
I am a little unfamilar with the `??` syntax, any reference that I can refer to to find the syntax